tests/filter.json

```diff
@@ -3673,6 +3673,52 @@
         "case"
       ]
     },
+    {
+      "name": "quoted True, double quotes",
+      "selector": "$[?@.a==\"True\"]",
+      "document": [
+        {
+          "a": "True"
+        },
+        {
```

Comment:
Would it make sense to add an entry like
```json
{
  "a": true
}
```
and similar ones to the other tests?

Reply:
Thank you!